snuba/snuba_migrations/search_issues/0001_search_issues.py

```diff
@@ -0,0 +1,97 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Array,
+    Column,
+    DateTime,
+    IPv4,
+    IPv6,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("group_id", UInt(64)),
+    Column("search_title", String()),
+    Column("fingerprint", Array(String())),
+    Column("occurrence_id", UUID()),
+    Column("occurrence_type_id", UInt(8)),
+    Column("detection_timestamp", DateTime()),
+    Column("event_id", UUID(Modifiers(nullable=True))),
+    Column("trace_id", UUID(Modifiers(nullable=True))),
+    Column("platform", String(Modifiers(low_cardinality=True))),
+    Column("environment", String(Modifiers(low_cardinality=True, nullable=True))),
+    Column("release", String(Modifiers(low_cardinality=True, nullable=True))),
+    Column("dist", String(Modifiers(low_cardinality=True, nullable=True))),
+    Column("receive_timestamp", DateTime()),
```

Comment:
The `detection_timestamp` would be the time when a vertical team processed an event and detected an issue, whereas `receive_timestamp` would be the event was ingested. Practically, the time difference between the two should be minimal since detection would likely happen close to when the event was ingested.